homeassistant/components/light/hyperion.py

```diff
@@ -28,7 +28,7 @@
 PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
     vol.Required(CONF_HOST): cv.string,
     vol.Required(CONF_PORT, default=DEFAULT_PORT): cv.port,
-    vol.Optional(CONF_DEFAULT_COLOR, default=DEFAULT_COLOR): cv.string,
+    vol.Optional(CONF_DEFAULT_COLOR, default=DEFAULT_COLOR): cv.ensure_list,
```

Comment:
Hi @balloob,

Maybe we are missunderstanding each other.
This is not my code, I have nearly made a small amendments to the value the script looks for (as above).
The current version of the script, that is distributes with all versions dosnt allow you to set that parameter and continue to use the switch.

As mentions about, I'm new to python, so if you have supplied an example, I'm not sure how to interpret it and where to place it in the script to avoid breaking it further.

If what you have supplied can be placed in the script and any point after the parameter is set, I can do that.

Happy to help where I can

Thank you

Joe